src/tree_sortable.rs

```diff
@@ -0,0 +1,106 @@
+// This file was generated by gir (17af302) from gir-files (11e0e6d)
+// DO NOT EDIT
+
+use SortType;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use auto::traits::TreeSortableExt as Auto;
+use std::thread;
+use std::cell::RefCell;
+use std::mem::transmute;
+use std::process;
+
+use glib_ffi::gpointer;
+use {TreeSortable, TreeIter};
+use ffi::{GtkTreeIter, GtkTreeSortable};
+
+pub trait TreeSortableExt {
+    // -- manual --
+    fn set_default_sort_func<F>(&self, sort_func: F)
+    where F: FnMut(&TreeSortable, &TreeIter, &TreeIter) -> i32 + 'static;
```

Comment:
Oh right, I totally forgot to use `cmp::ordering`...

However, isn't it better to let the user handles the value he gets back? By returning the type he wants, it means we have to use `get_value` directly in our function (or maybe I misunderstood something?). If he wants to get back `TreeIter`, I don't see how we could do it.